docs/contents/releases/1-20/2/release-announcement.txt

```diff
@@ -0,0 +1 @@
+Amazon EKS Distro v1.20-2 is now available. This release now supports Kubernetes v1.20.7. Additional highlights of this release include the latest version of Amazon Linux 2 and a patch to fix a bug related to AWS TargetGroup. Amazon EKS Distro builds of Kubernetes 1.20-2 are available through ECR Public Gallery (https://gallery.ecr.aws/?searchTerm=EKS+Distro) and GitHub (https://github.com/aws/eks-distro)
```

Comment:
Wouldn't it be more semantically correct to say Amazon EKS Distro 1.20-2 builds, since Kubernetes 1.20-2 might be misleading. Basically tying the release branch and number to EKS-D instead of Kubernetes itself.

Reply:
I'm a little confused. v1.20-2 is the EKS-D version.

Reply:
Oh wait. The last sentence. Yeah I can see that, I'll revise it